src/Text/Pandoc/Templates.hs

```diff
@@ -90,6 +90,7 @@ getDefaultTemplate writer = do
        "docbook" -> getDefaultTemplate "docbook5"
        "epub"    -> getDefaultTemplate "epub3"
        "beamer"  -> getDefaultTemplate "latex"
+       "jats"    -> getDefaultTemplate "jats_archiving"
```

Comment:
Don't we need `jats_archiving` and the others here as well?

Reply:
The templates for those have the same name as the format specifier. So they are handled by the default case a bit further down.

Reply:
Ah yes, I see.